### DISCUSSION TITLE
`make lint` failed and got error

### DISCUSSION THREAD

**Discussion by hantmac** [databendlabs/databend#5469] [Q&A] [answered]
**Summary**
I clone the code and `make lint` in my local env to fmt , got the flowing error:

```
yapf -ri tests/
yapf: toml package is needed for using pyproject.toml as a configuration file
make: *** [lint] Error 1
```

Then I found [this issue](https://github.com/google/yapf/issues/936), maybe we should move `.style.yapf` to the `tests` dir?

**Comment by Xuanwo** [comment, marked as answer]
Please try:

```shell
pushd tests && poetry shell && popd
```

**Reply by hantmac** [reply]
@Xuanwo  Thanks for your help.

<img width="989" alt="image" src="https://user-images.githubusercontent.com/7600925/169275604-40ab15ff-1f02-4879-9a6a-bba65f142e39.png">

Then got the same error:

<img width="982" alt="image" src="https://user-images.githubusercontent.com/7600925/169275792-a7b70270-700d-4a23-a704-05e91ef992a9.png">

**Reply by Xuanwo** [reply]
Oh, you are right. We are missing the toml deps.

**Reply by hantmac** [reply]
> Oh, you are right. We are missing the toml deps.

That's it.

**Reply by Xuanwo** [reply]
Should be fixed in https://github.com/datafuselabs/databend/pull/5470

**Reply by hantmac** [reply]
> Should be fixed in #5470

Fixed.
